public/index.html

```diff
@@ -0,0 +1,104 @@
+           <a href="#">[ Read More ]</a>
+         </article>
+         <article class="content">
+           <h3>Ray Barbee Talks 'Ban This'</h3>
+           <h4>October 04, 2013</h4>
+           <p>Since filming is in full swing for the Vans video Adventures With Chris has taken the oppertunity to sit some of the team down to look behind the skating of some of thier earlier parts. Today we sit down Ray Barbee, one of the smoothest guys to ever step on a skateboard and Chris' childhood idol...</p>
+           <a href="#">[ Read More ]</a>
+         </article>
+        </section>
+        <button class="the arrow"><span>SKATEBOARDERS</span></button>
+        <div class="img-grid">
+          <div class="img-grid__sec">
+          <h1>LANCE MOUNTAIN</h1>
+          <img class="img-small" src="img/sk8er-4.png"/>
+        </div>
+          <div class="img-grid__sec">
+          <h1>RODNEY MULLEN</h1>
+          <img class="img-small" src="img/sk8er-3.png"/>
+        </div>
+          <div class="img-grid__sec">
+          <h1>TOMMY GUERRERO</h1>
+          <img class="img-tall" src="img/sk8er-2.png"/>
+        </div>
+          <div class="img-grid__sec">
+          <h1>STEVE CABALLERO</h1>
+          <img class="img-long" src="img/sk8er-1.png"/>
+        </div>
+        </div>
+        <div class="skateboard-container">
+          <img class="skateboard" src="img/sk8-3.png"/>
+          <img class="skateboard" src="img/sk8-4.png"/>
+          <img class="skateboard" src="img/sk8-5.png"/>
+          <img class="skateboard" src="img/sk8-6.png"/>
+          <img class="skateboard" src="img/sk8-7.png"/>
+          <img class="skateboard" src="img/sk8-8.png"/>
+          <img class="skateboard" src="img/sk8-9.png"/>
+          <img class="skateboard" src="img/sk8-10.png"/>
+          <img class="skateboard" src="img/sk8-2.png"/>
+          <img class="skateboard" src="img/sk8-1.png"/>
+        </div>
+
+        <div class="footer-container">
+          <h3 class="footer-head">Contact Us For More Information</h3>
+          <p class="footer-context">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+          <i class="fa fa-facebook-official"></i>
+          <i class="fa fa-rss-square"></i>
+          <i class="fa fa-twitter-square"></i>
+          <i class="fa fa-youtube-square"></i>
+        </div>
+        <input type="Text" name="Name" value="Name">
```

Comment:
These inputs should be in your footer. But I recommend that in the footer you have dividers for the left and right hand side. Also don't forget the submit button!